docs/testimonials.md

```diff
@@ -0,0 +1,97 @@
+# Testimonials
+
+You might not be ready to make the jump to Jujutsu yet. That's OK — in order
+to provide some motivation, we've collected a number of valuable testimonials 
+from both users and developers, which are sure to tip the scales and manipulate
+you in our favor!
+
+## What the users have to say 
+
+> I've spent many years of my career working on version control. What I like most
+> about Jujutsu is how it has non-obvious solutions to UX problems that we've run
+> into in the past. Most people may not realize this at first, but the many novel
+> features in Jujutsu all interlock to make it easy to use.
+> 
+> For example, consider Jujutsu's support for automatically rebasing descendants 
+> of amended revisions. When we implemented that in Mercurial, we ran into an 
+> issue: what if there's a merge conflict? Our solution was to warn users if that
+> occurred, and just not perform the auto-rebase. Now, suddenly, users have to 
+> understand that there can be old versions of the same revision visible in their
+> log. They also need to learn how to get themselves out of this state.
+> 
+> In contrast, Jujutsu's solution is to simply make merge conflicts first-class.
+> This is not just an improvement in general, it is also specifically an
+> improvement for auto-rebase—users no longer have to learn about old 
+> versions of a revision unless they want to look at the obslog.
+> 
+> Over and over, I'm struck by how well Jujutsu demonstrates this kind of 
+> evolved thinking, As an experienced version control developer, 
+> I deeply appreciate it.
+
+Rain, engineer at Oxide and former VCS developer
+
+> Jujutsu is pretty cool, you can even keep most of your existing workflows 
+
+Ben, who really doesn't want you keeping your existing workflow.
+
+> Wait, it's not called Jujitsu?
+
+Phil, a Mercurial contributor and user.
+
+> It's the easiest time I've ever had learning a tool this deeply this quickly, 
+> because of the ability to experiment and undo, instead of triple-checking 
+> before trying a new scary command.
+
+Scott Olson, a pretty advanced Git user who replaced it completely with 
+Jujutsu.
+
+> Jujutsu is amazing because it trivializes some of the things I couldn't have come 
+> up with anything remotely as elegant. 
+> It's so rare that a solution attacks the innermost core of a problem so 
+> thoroughly, I genuinely feel blessed to be in its presence. And also a bit 
+> vindicated in not even trying to learn to use any of the tools that felt like
+> more crutches stacked upon a sand castle  
+
+An anonymous user.
+
+> only breaks half as often as my other tools!
+
+jyn, ex-rustc bootstrap developer, a place where tools break down all the time.
+
+> I initially started to use Jujutsu for personal repos where it has quickly 
+> gone from "neat, let's try this more" to "very neat, added to my permanent
+> config and automatically installed for new machines". 
+
+Poliorcetics
```

Comment:
@poliorcetics, thanks for the testimonial, is this good for you?

Reply:
English question (not a native speaker): is it better to say `personal repos where it has` or `personal repos and it has` ? I'm unsure about the differences and I used `where` by habit I think

Reply:
i would either say "personal repos, where" or "personal repos and". "personal repos where" without a comma looks odd to my eye.

Reply:
I think either gets the point across, though I would personally use `and it has`, and also add a comma:

`Jujutsu for personal repos, and it has`

But that's just my stylistic choice. It's completely fine as-is.

Reply:
Then @PhilipMetzger `, and` or `, where` are fine :)

Reply:
I've added the comma (`, and`) and reverted it to the fully spelled out version.

Reply:
Yes thanks :)

Reply:
No, thank you for your contribution :-).